package.json

```diff
@@ -6,10 +6,9 @@
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
     "test": "jest",
-    "prettier-js": "prettier --write '{pages,src}/**/*.{js,jsx}'",
-    "prettier-md": "prettier --write 'static/docs/**/*.md'",
+    "start": "NODE_ENV=production node server.js",
+    "pretty": "pretty-quick --staged --no-restage --bail",
```

Comment:
As noted in https://github.com/iterative/dvc.org/pull/647#issuecomment-538110156, I realized we have 2 scripts for similar purposes: `yarn pretty` (uses `pretty-quick`) and `yarn lint` (uses `eslint`). Both are enforced locally with a Git pre-commit hook (see line 71 of this same file) but this could be disabled by the developer. CircleCI only enforces the latter, `yarn lint`.

Is this correct? Seems strange to me @shcheklein and @iAdramelk.

Reply:
p.s. I cleaned up some old linting scripts as you can see in e74cbc1.